packages/ps/src/commands/ps/wait.ts

```diff
@@ -0,0 +1,80 @@
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {cli} from 'cli-ux'
+
+async function delay(millis: number) {
+  return new Promise(resolve => setTimeout(resolve, millis))
+}
+
+export default class Wait extends Command {
+  static description = 'wait for a release to cycle in'
+  static help = `
+When a release is created, it may take a while for all dynos to be
+running the new version. This is especially true for applications in
+Heroku Private Spaces or using the Common Runtime preboot feature,
+where dynos cycle in gradually when a new release is deployed. This
+command allows you to wait until all dynos are on the latest release
+version.
+  `
+  static flags = {
+    app: flags.app({required: true}),
+    remote: flags.remote(),
+    type: flags.string({char: 't', description: 'wait for one specific dyno type'}),
+    'wait-interval': flags.integer({char: 'w', description: 'how frequently to poll in seconds (to avoid rate limiting)'}),
+    'with-run': flags.boolean({char: 'R', description: 'whether to wait for one-off run dynos'})
+  }
+
+  async run() {
+    const {flags} = this.parse(Wait)
+
+    if (flags['with-run'] && flags.type) {
+      throw new Error('Cannot specify both --type and --with-run')
+    }
+
+    const releases = await this.heroku.request<any[]>(`/apps/${flags.app}/releases`, {
+      partial: true,
+      headers: {
+        Range: 'version ..; max=1, order=desc'
+      }
+    })
+
+    if (releases.body.length === 0) {
+      throw new Error(`App ${flags.app} has no releases`)
+    }
+
+    const latestRelease = releases.body[0]
+
+    let released = true
+    let interval = flags['wait-interval']
+    if (!interval || interval < 0) {
+      interval = 10
+    }
+
+    let lolLint = true
+    while (lolLint) {
```

Comment:
The linter complained about `while (true)` but I think the flow through the loop makes more sense like that. Obviously this is not a good hack around it, but should we disable linter for this line? Or is there a better way to organize this logic that I am missing?

Reply:
you might be able to do `while (1 as any)`